jobboerse/_posts/2021-01-19-verfassungsschutz.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Stellenangebote beim Bundesamt für Verfassungsschutz
+author: Henning
+---
+
+Es gibt passende Stellenangebote für Physiker beim Bundesamt für Verfassungsschutz.
```

Comment:
Gendern bitte :D mit einer Schreibweise deiner Wahl.

Reply:
Hätte ich dran denken können... War nur copy and paste. Ist jetzt korrigiert